benefits/static/css/styles.css

```diff
@@ -82,6 +82,10 @@ main {
   flex-shrink: 0 !important;
 }
 
+main#main-content.main-content {
```

Comment:
if there is already an id selector `#main-content` do we also need the class selector `.main-content`?

Reply:
I wrote the style declaration like this because that's what the HTML element has, it has both the ID and the class already.